x-pack/plugins/alerting/server/rules_client/rules_client.ts

```diff
@@ -990,6 +1000,9 @@ export class RulesClient {
       snoozed: snoozedBuckets.reduce((acc, bucket) => acc + bucket.doc_count, 0),
     };
 
+    const tagsBuckets = resp.aggregations.tags.buckets;
```

Comment:
Are we confident the `buckets` are always there?
ES has a tendency to drop keys in aggs when there are no hits (such as a cluster with no rules in it or when a filter is applied).

Worth treading with caution and checking for undefined keys.

Reply:
Always build these things defensively, otherwise the UI breaks and we have no idea unless someone files an SDH.

Thanks 🙏